en-US/firefox/sync.lang

```diff
@@ -1,42 +1,67 @@
 ## firefox_sync_mobile_buttons_update ##
-## firefox_sync_non_fx ##
 ## fx38_sync_updates ##
 ## ios_announce ##
-## firefox_title_meta_bug1345338 ##
+## sync-update-q3-2017 ##
 ## NOTE: url for demo server is: https://www-dev.allizom.org/firefox/sync/
 ## URL: https://www-dev.allizom.org/%LOCALE%/firefox/sync/
 
 
-## TAG: firefox_title_meta_bug1345338
 # HTML page title
 ;Use Bookmarks, Tabs and Passwords Across Devices | Firefox
 Use Bookmarks, Tabs and Passwords Across Devices | Firefox
 
 
-# String is obsolete. Do not remove.
-# HTML page title
-;Keep your Firefox in sync
-Keep your Firefox in sync
-
-
-## TAG: firefox_title_meta_bug1345338
 # HTML page description
 ;Bring your Firefox bookmarks, tabs, or passwords wherever you go with Firefox Sync. Share between computers and mobile devices. Get Firefox now!
 Bring your Firefox bookmarks, tabs, or passwords wherever you go with Firefox Sync. Share between computers and mobile devices. Get Firefox now!
 
 
+## TAG: sync-update-q3-2017
+;Browse boldly. Connect your devices.
+Browse boldly. Connect your devices.
+
+
+## TAG: sync-update-q3-2017
+;Add Firefox on your personal devices to get your bookmarks, open tabs and passwords anywhere. Get secure access to everything you need when you sign into your Firefox Account.
+Add Firefox on your personal devices to get your bookmarks, open tabs and passwords anywhere. Get secure access to everything you need when you sign into your Firefox Account.
+
+
+# String is obsolete. Do NOT remove.
 ;Ready, Set, Sync
 Ready, Set, Sync
 
 
+# String is obsolete. Do NOT remove.
 ;You’re signed up and ready to access <em>your</em> Firefox anywhere, anytime.
 You’re signed up and ready to access <em>your</em> Firefox anywhere, anytime.
 
 
+## TAG: sync-update-q3-2017
+;Your web, how you like it, everywhere
+Your web, how you like it, everywhere
+
+
+## TAG: sync-update-q3-2017
+;Sign into your Firefox Account to access your protected passwords, open tabs and bookmarks. Add Firefox to your personal devices and securely connect your browsing experience everywhere. Keep everything set up how you like it, wherever you go.
+Sign into your Firefox Account to access your protected passwords, open tabs and bookmarks. Add Firefox to your personal devices and securely connect your browsing experience everywhere. Keep everything set up how you like it, wherever you go.
+
+
+## TAG: sync-update-q3-2017
+;If you lived here you’d be home by now
+If you lived here you’d be home by now
```

Comment:
I added a long line of explanation. If you think it is good as is, we can move forward with it. Otherwise, we can wait for comments in the bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1380490